.github/workflows/release-udp-exporter.yml

```diff
@@ -0,0 +1,76 @@
+name: Release ADOT OTLP UDP Exporter
+
+on:
+  push:
+    branches:
+      - "udp-*"
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version number for deployment e.g. 0.0.1'
+        required: true
+        type: string
+
+jobs:
+  build-test-publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Java
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'  # or your preferred version
+          distribution: 'temurin'
+          cache: 'gradle'    # Add gradle caching
+
+      - name: Build, Test and Publish UDP exporter
+        working-directory: exporters/aws-opentelemetry-xray-udp-span-exporter
+        run: |
+          gradle build 
+          gradle publishToMavenLocal
+
+      - name: Download and run X-Ray Daemon
+        run: |
+          mkdir xray-daemon
+          cd xray-daemon
+          wget https://s3.us-west-2.amazonaws.com/aws-xray-assets.us-west-2/xray-daemon/aws-xray-daemon-linux-3.x.zip
+          unzip aws-xray-daemon-linux-3.x.zip
+          ./xray -o -n us-west-2 -f ./daemon-logs.log --log-level debug &
+
+      - name: Run Sample App in Background
```

Comment:
Before running the sample app, there should be a step to install the version of UDP Exporter that was published via `publishToMavenLocal`.

Reply:
Fixed in: https://github.com/aws-observability/aws-otel-java-instrumentation/pull/1049/commits/5d5c408a952715b84c84bd5981b28121ca914475